docs/install.md

```diff
@@ -107,26 +114,28 @@ daemonset.apps/csi-hostpathplugin created
 1. Run the following command
 
 ```bash
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/00-namespace.yaml
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/0000_10_sharedresource.crd.yaml
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/01-service-account.yaml 
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/02-cluster-role.yaml
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/03-cluster-role-binding.yaml
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/csi-hostpath-driverinfo.yaml
-oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.8/deploy/csi-hostpath-plugin.yaml 
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/00-namespace.yaml
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/0000_10_sharedconfigmap.crd.yaml
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/0000_10_sharedsecret.crd.yaml
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/01-service-account.yaml 
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/02-cluster-role.yaml
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/03-cluster-role-binding.yaml
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/csi-hostpath-driverinfo.yaml
+oc apply -f --filename https://raw.githubusercontent.com/openshift/csi-driver-shared-resource/release-4.10/deploy/csi-hostpath-plugin.yaml 
```

Comment:
Should we replace `release-4.10` here with `master` instead?

Reply:
I don't think so, as master could be unstable, once a given release goes out.

Reply:
So we will have to update this readme every release?

Reply:
lesser of the evils IMO ... at least for now ... and if it lags, it is not the worse thing ... and it is easy enough for someone to figure our they can tweak that to another branch

possible this just goes away or morphs into a "dev only" mode at some point, in which case we switch to master 

I'd rather hold off on that while we are in this intermediate state